Sources/Apodini/Response/Blob.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  Blob.swift
+//  
+//
+//  Created by Paul Schmiedmayer on 5/26/21.
+//
+
+import NIO
+import Foundation
+import NIOFoundationCompat
+
+
+/// A binary large object (blob) that can be used to return binary data from a `Handler`
+public struct Blob: Encodable, ResponseTransformable {
+    enum CodingKeys: String, CodingKey {
+        case byteBuffer
+        case type
+    }
+    
+    
+    /// The `ByteBuffer` representation of the `Blob`
+    public let byteBuffer: ByteBuffer
+    /// The MIME type associated with the `Blob`
+    public let type: MimeType?
+    
+    
+    /// - Parameters:
+    ///   - data: The `Data` representation of the `Blob`
+    ///   - type: The MIME type associated with the `Blob`
+    public init(_ data: Data, type: MimeType? = nil) {
+        self.byteBuffer = ByteBuffer(data: data)
+        self.type = type
+    }
+    
+    
+    /// - Parameters:
+    ///   - byteBuffer: The `ByteBuffer` representation of the `Blob`
+    ///   - type: The MIME type associated with the `Blob`
+    public init(_ byteBuffer: ByteBuffer, type: MimeType? = nil) {
+        self.byteBuffer = byteBuffer
+        self.type = type
+    }
+    
+    
+    public func encode(to encoder: Encoder) throws {
+        print(
+            """
+            Information: The used Exporter currently doesn't support Blob's as a content of an Apodini Reponse.
```

Comment:
Maybe we should wrap this in a if DEBUG? Otherwise this could cause quite some spam on production systems, especially since it is not explicitly _wrong_ for an exporter to use this strategy. It would also be great if we could use the Application’s Logger here…should we provide a static property on Application, which always references rhe last Application that was initialized?

Reply:
Good idea, I have added a global logger as a static property of the `Application`